script.js

```diff
@@ -1,4 +1,81 @@
+
+// Define global variables
+var gamesPlayed = -1;
+var userWinCount = 0;
+var currentGameMode = 'waiting for user name'; 
+var computerObject = '';
+
+// Function to randomly generate a number which corresponds to either Scissors, Paper or Stone and return this object.
+function chooseRandomObject () {
+  var randomObject = Math.floor(Math.random()*3+1);
+  if (randomObject == 1) {
+    computerObject = 'Scissors';
+  }
+  else if (randomObject == 2) {
+    computerObject = 'Paper';
+  } else {
+    computerObject = 'Stone';
+  };
+  return computerObject;
+};
+
+
+// SPS logic 
+function playGame (userName, userGuess) {
```

Comment:
Please develop your program with ESLint to automatically standardise formatting. Inconsistent formatting is unacceptable in industry and you will lose points and/or your job as a SWE.